### ISSUE TITLE
Docs need to be updated

### ISSUE THREAD

**Issue by Isaac-Tait** [decaporg/gatsby-plugin-decap-cms#1]
FYI - [The docs](https://decapcms.org/docs/gatsby/#add-decap-cms-to-your-site) still state you need to run the following `npm install --save decap-cms-app gatsby-plugin-netlify-cms`

**Comment by martinjagodic** [comment]
True. Docs will be updated once we release 3.1, which is currently in beta.
